forge-gui/res/cardsfolder/f/firestorm.txt

```diff
@@ -3,8 +3,8 @@ ManaCost:R
 Types:Instant
 A:SP$ DealDamage | Cost$ R Discard<X/Card> | ValidTgts$ Any | TargetMin$ 0 | TargetMax$ MaxTargets | NumDmg$ X | SpellDescription$ CARDNAME deals X damage to each of X targets.
 SVar:X:TargetedObjects$Amount
-SVar:MaxTargets:SVar$MaxPlayers/Plus.MaxCreaturesAndPlaneswalkers
+SVar:MaxTargets:SVar$MaxPlayers/Plus.MaxPermanents
 SVar:MaxPlayers:PlayerCountPlayers$Amount
-SVar:MaxCreaturesAndPlaneswalkers:Count$Valid Creature,Planeswalker
+SVar:MaxPermanents:Count$Valid Creature,Planeswalker
```

Comment:
Valid Any

Reply:
Took the opportunity to standardize other relevant scripts. Meteor Shower seemed to have one step too many, so hopefully I trimmed it reasonably.